### DISCUSSION TITLE
cn/2024/09/29/chi2/

### DISCUSSION THREAD

**Discussion by giscus** [hongtaoh/hongtaoh.github.io#183] [Announcements]
# cn/2024/09/29/chi2/

https://hongtaoh.com/cn/2024/09/29/chi2/

**Comment by earfanfan** [comment]
>取无数个样本。

这一句莫名戳中了我的笑点，哈哈，我记得形容样本数都是说有限或无限。

**Reply by hongtaoh** [reply]
我比较喜欢口语化。

**Comment by earfanfan** [comment]
安利一个与主题无关的游戏：<https://ncase.me/trust/>。
